packages/admin/src/form/FinalFormSearchTextField.tsx

```diff
@@ -0,0 +1,38 @@
+import { InputAdornment, TextField } from "@material-ui/core";
+import { TextFieldProps } from "@material-ui/core/TextField";
+import * as React from "react";
+import { FieldRenderProps } from "react-final-form";
+import { useIntl } from "react-intl";
+
+import { useStyles, useThemeProps } from "./FinalFormSearchTextField.styles";
+
+export const FinalFormSearchTextField: React.FunctionComponent<TextFieldProps & FieldRenderProps<string, HTMLInputElement | HTMLTextAreaElement>> = ({
+    icon,
+    input: { name, onChange, value, ...restInput },
+    placeholder,
+    variant = "outlined",
+    ...restProps
+}) => {
+    const intl = useIntl();
+    const themeProps = useThemeProps();
+    const classes = useStyles({});
+
+    return (
+        <TextField
+            {...restProps}
+            name={name}
+            variant={variant}
+            onChange={onChange}
+            value={value}
+            placeholder={placeholder ?? intl.formatMessage({ id: "comet.finalformsearchtextfield.default.placeholder", defaultMessage: "Search" })}
+            InputProps={{
```

Comment:
Why is `InputProps` used, normally we use `inputProps`  https://github.com/vivid-planet/comet-admin/blob/master/packages/admin/src/form/Radio.tsx#L11?

The docs https://material-ui.com/api/text-field/ say:
> inputProps:  Attributes applied to the input element.
> InputProps: Props applied to the Input element.

🤔 
In fact, both types partially overlap.

Reply:
@rabengraph i don't really know what's the difference. Why do we "normally" only use `inputProps`? I didn't even know that we normally use that?

The reason i use `InputProps` here is the startAdornment. This does not work on `inputProps`

Reply:
when i use it this way, it works, but for me it doesn't make it any better.


<img width="1369" alt="Screenshot 2021-05-19 at 14 20 37" src="https://user-images.githubusercontent.com/6098356/118811502-62b7d580-b8ad-11eb-8fc6-c2b3424f799a.png">